java/ql/src/Security/CWE/CWE-295/AndroidMissingCertificatePinning1.java

```diff
@@ -0,0 +1,2 @@
+// BAD - By default, this network call does not use certificate pinning
+URLConnection conn = new URL("https://example.com").openonnection();
```

Comment:
```suggestion
URLConnection conn = new URL("https://example.com").openConnection();
```
I believe this is a typo

Reply:
Good catch, thanks!